GPflow/likelihoods.py

```diff
@@ -446,7 +446,7 @@ def predict_density(self, Fmu, Fvar, Y):
         if isinstance(self.invlink, RobustMax):
             gh_x, gh_w = hermgauss(self.num_gauss_hermite_points)
             p = self.invlink.prob_is_largest(Y, Fmu, Fvar, gh_x, gh_w)
-            return p * (1. - self.invlink.epsilon) + (1. - p) * self.invlink._eps_K1
+            return p * np.log1p(- self.invlink.epsilon) + (1. - p) * np.log(self.invlink._eps_K1)
```

Comment:
I'm not sure this is right. I think maybe it should be 
```python
tf.log(p * (1. - self.invlink.epsilon) + (1. - p) * self.invlink._eps_K1)
```
i.e. with the log outside the expectation. Happy to be corrected if wrong.